trunk/src/hpinterp/update_gdof.F90

```diff
@@ -62,7 +57,15 @@ subroutine update_gdof()
    integer, allocatable :: nod_loc(:),nod_tmp(:),nod_glb(:),nod_rnk(:)
    real(8), dimension(:,:), pointer :: buf
 !
+!..use threaded routine; not recommended without proper verification first
+   logical :: USE_THREADED = .false.
```

Comment:
unlike `update_Ddof`, the `update_gdof` routine is user-independent. Is this just not tested enough yet to be certain is works w/o issues?

Reply:
It's worked well for my problems; I typically disabled it along with `update_Ddof` in the laser problem so I'm not sure about that case. I haven't tested it on a ton of geometries though.

Reply:
Ok, we can do that in `next`. For sure, it'll good idea to test with constrained nodes and different types of physics variables.